components/Home/Home.vue

```diff
@@ -71,6 +73,66 @@ const device = useDevice()
 //
 // Hooks
 //
+onBeforeMount(async () => {
+  const favs = getHashPart(router, 'favs')
+  if (favs) {
+    try {
+      const addressHashes: string[] = []
+
+      favs
+        .split(',')
+        .forEach((id) => {
+          if (!Number.isNaN(Number(id)))
+            favoritesStore.toggleFavorite(Number(id))
+
+          if (id.startsWith('addr-'))
+            addressHashes.push(id.substring(5))
+        })
+
+      await Promise.all(addressHashes.map(async (hash) => {
+        const coordinates = decodeBase32(hash)
+        const address = await $fetch<ApiSearchResult<ApiAddrSearchResult>>(`${config!.API_ADDR}/reverse?lon=${coordinates.lng}&lat=${coordinates.lat}`)
+
+        if (!address || !address.features.length)
+          throw createError({ statusCode: 404, message: `Error while reverse geocoding: ${hash}` })
```

Comment:
This exception stop all ?
If we cannot do the reverse, we can continue anyway.

Reply:
mmh I think it just handles the error and throws it but it's non-fatal.